drivers/dma/dma_mcux_lpc.c

```diff
@@ -642,26 +702,11 @@ static int dma_mcux_lpc_reload(const struct device *dev, uint32_t channel,
 	struct dma_mcux_lpc_dma_data *dev_data = dev->data;
 	int8_t virtual_channel = dev_data->channel_index[channel];
 	struct channel_data *data = DEV_CHANNEL_DATA(dev, virtual_channel);
-	uint8_t src_inc, dst_inc;
 	uint32_t xfer_config = 0U;
 
-	switch (data->dir) {
-	case MEMORY_TO_MEMORY:
-		src_inc = 1;
-		dst_inc = 1;
-		break;
-	case MEMORY_TO_PERIPHERAL:
-		src_inc = 1;
-		dst_inc = 0;
-		break;
-	case PERIPHERAL_TO_MEMORY:
-		src_inc = 0;
```

Comment:
I am a little confused how we are handling peripheral to memory and vice versa now?

Reply:
We would always increment by `1 x width`.